pkg/util/util.go

```diff
@@ -284,6 +287,22 @@ func HasNodeAffinityChanged(
 	return !reflect.DeepEqual(cluster.Spec.Placement.NodeAffinity, existingAffinity.NodeAffinity)
 }
 
+// HasSchedulerStateChanged checks if the stork has been enabled/disabled in the StorageCluster
+func HasSchedulerStateChanged(
+	cluster *corev1.StorageCluster,
+	previousSchedulerName string,
+) bool {
+	if cluster.Spec.Stork == nil {
+		return false
```

Comment:
If stork was enabled before and then the spec was set to empty, this should return true.

Reply:
Can you add UTs for this function?

Reply:
Done